### DISCUSSION TITLE
有些网站域名不能访问

### DISCUSSION THREAD

**Discussion by jwkinko** [vernesong/OpenClash#2626] [Q&A]
Redir-Host兼容模式下

取消 dns设置下的
*远程域名解析
追加上游 DNS
追加默认 DNS

并且删除配置文件的dns 项
之前的域名就能访问，其他网站都正常访问。
有大老解释下openclash 的dns 解析流程马？

**Comment by vernesong** [comment]
尝试设置fallback dns和fallback-filter
